test/functional/helpers.lua

```diff
@@ -417,8 +418,12 @@ local function curtab(method, ...)
   return tabpage(method, 0, ...)
 end
 
-local function expect(contents)
-  return eq(dedent(contents), curbuf_contents())
+local function expect(contents, any)
```

Comment:
it does help, I use it also for https://github.com/neovim/neovim/pull/5878 (I finished that PR but haven't pushed it yet). I made it expect_any.